lanehunter-dev/refactor-tractor
# Feature/recipes to cook

#### What's this PR do?
- This PR adds functionality to add recipes to a new section: "Recipes to Cook"
- The 'chef hat' icon in the 'My Recipes' section and the 'apple icon' in each recipe card has been changed to a heart icon to make it more clear to the use that this is a 'Favorite' button/section.
#### Where should the reviewer start?
Main files effected were scripts.js, index.html, banner.scss, header.scss, domUpdates.js and new asset additions in the /images directory.
#### How should this be manually tested?
- Load the page and add items to the "Recipes to Cook" section.
- Click the mixer icon up top
- Ensure the recipe was added to the page
- Remove items from the RTC page
- Ensure they have been removed
- Repeat this process from the favorite recipes page
#### Questions:
